converter/internal/prometheusconvert/testdata/discovery_relabel.yaml

```diff
@@ -0,0 +1,26 @@
+scrape_configs:
+  - job_name: "prometheus1"
+    static_configs:
+      - targets: ["localhost:9090"]
+    relabel_configs:
+      - source_labels: [__address1__]
+        target_label: __param_target1
+      - source_labels: [__address2__]
+        target_label: __param_target2
```

Comment:
It'd be great if we could include a relabel rule with a regular expression which uses escaped characters, especially an escaped double quote. 

I'd like to verify that it won't generate an invalid River string, and that the escaping is retained.